appengine/flexible_python37_and_earlier/tasks/README.md

```diff
@@ -103,13 +103,19 @@ the "name" value (for instance, if the name is
 "projects/my-project/locations/us-central1/queues/my-appengine-queue", then the
 location is "us-central1").
 
-```
+```sh
 export LOCATION_ID=us-central1
 ```
+
 ### Using App Engine Queues
+
 Running the sample will create a task, targeted at the `/example_task_handler`
 endpoint, with a payload specified:
 
-```
+> **Note**
+> Please update `create_app_engine_queue_task.py` before running the following
+> command.
```

Comment:
I have mixed feelings on how it looks in the README: https://github.com/XinruZhang/python-docs-samples/tree/cloud-task-example/appengine/flexible_python37_and_earlier/tasks

fine to leave as is though, maybe it's just me 😅

Reply:
Thank you Dan @dandhlee! 

> I have mixed feelings on how it looks in the README: https://github.com/XinruZhang/python-docs-samples/tree/cloud-task-example/appengine/flexible_python37_and_earlier/tasks

I'm having trouble understanding why do you have the mixed feelings 🤔 Would you mind elaborating a bit? Ty!

Reply:
Ah sorry - I think it looks a bit cramped with all of them in one line. perhaps if you were to put it like this:

```
Note 
Please update create_app_engine_queue_task.py before running the following command.
```
would be better I think! But, it's very minor so fine to leave as is 😄